storage/database/schema.go

```diff
@@ -185,3 +188,7 @@ func governanceKey(num uint64) []byte {
 	binary.LittleEndian.PutUint64(b, num)
 	return append(governancePrefix[:], b[:]...)
 }
+
+func databaseDirKey(dbEntryType uint64) []byte {
```

Comment:
For dbEntryType, uint64 is needed? It can be covered by small uint?

Reply:
It can be, something like `uint16`, however, maximum number of entries stored by this function is the number of total database partitions, which means less than 10. Maybe we can use `uint16` but what I wanted to use already existing number-encoding logic.